install/yaml/install.yaml

```diff
@@ -48,13 +48,13 @@ rules:
   resources: ["customresourcedefinitions"]
   verbs: ["get"]
 - apiGroups: ["stable.agones.dev"]
-  resources: ["gameservers", "gameserversets"]
+  resources: ["gameservers", "gameserversets", "gameserversets/status"]
```

Comment:
Is there opportunity here to get more restrictive with our rbac permissions?

I mean - does the controller need to be able to update a gameserverset for example?

Reply:
I will try to restrict them

Reply:
@markmandel I have restricted new subresource permissions.
However cannot restrict `gameserversets` permissions. In fleets and fleetautoscalers controllers we are using main Resource endpoint `gsSetCopy.Spec.Replicas` and Spec.Replicas of the fleet:
https://github.com/GoogleCloudPlatform/agones/blob/0e92c90b9f0da86ab34cae53e74382113f56bf6e/pkg/fleets/controller.go#L268
https://github.com/GoogleCloudPlatform/agones/blob/0e92c90b9f0da86ab34cae53e74382113f56bf6e/pkg/fleetautoscalers/controller.go#L220